### DISCUSSION TITLE
Release 2.2.2

### DISCUSSION THREAD

**Discussion by olavopeixoto** [olavopeixoto/plugin.video.brplay#190] [Announcements]
fixed globoplay and globosat geofencing playback

<hr /><em>This discussion was created from the release <a href='https://github.com/olavopeixoto/plugin.video.brplay/releases/tag/2.2.2'>Release 2.2.2</a>.</em>

**Comment by marcusmuller** [comment]
Senti falta dos canais do interior dos estados que foram add. O que houve?

**Comment by olavopeixoto** [comment]
Nao fiz o merge ainda.

**Reply by marcusmuller** [reply]
Ok. Ficamos no aguardo do prox. update.

Detalhe importante:
Os canais globo internacional euro e Portugal, estão com bloqueio ainda de restrição geográfica. Faz ele funcionar sem o uso de VPN, como tá feito na Globo US. 

WLW.

**Reply by olavopeixoto** [reply]
Não tem nenhuma mágica ali. A API está bloqueando na Europa e não está nos EUA. Única forma é usar um proxy ou uma VPN.

**Comment by jwallopes** [comment]
Depois que o aplicativo NOW mudou de nome para ClaroTV, passou a dar erro  no addon. 
E o TNT nunca funcionou, nem nas versões anteriores.

**Comment by rogerioronin** [comment]
Parabéns por ter adicionado o VIX, é sempre bom ter opções gratuitas legais, se possível de uma olhada no Runtime https://www.runtime.tv, é gratuito também e possui 3 canais lineares (iguais a da Pluto TV).

**Comment by andreluigo** [comment]
Até algumas semanas está tudo funcionando perfeitamente...
Fiquei sem usar alguns dias e agora todo conteúdo que não seja Rede Glodo dá erro com uma msg de conteúdo apenas para assinantes (403 - VIDEO IS FOR SUBSCRIBERS ONLY. AUTHORIZATION REQUIRED.)
Assinatura está ok pq em outras plataformas abre normal!
